packages/examples/.storybook/main.ts

```diff
@@ -0,0 +1,12 @@
+import { defineMain } from "@storybook/react-vite/node";
+
+const config = defineMain({
+  stories: ["../stories/**/*.mdx", "../stories/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: ["@storybook/addon-docs"],
```

Comment:
We don't need to register this plugin as it's effectively just React components accessing the Storybook API.

Can you confirm @dev-bnguyen?

Reply:
I think that's just required when using addon-docs, since it's mentioned in [the Installation](https://www.npmjs.com/package/@storybook/addon-docs), and to write .mdx stories

Pulled it down and removed it to double check, and the stories no longer work
<img width="1577" height="458" alt="Screenshot 2025-07-24 at 12 42 58 pm" src="https://github.com/user-attachments/assets/849fbf85-0971-4ed5-933d-6a7fee02ac67" />

Reply:
Sorry, I'm referring to this addon not being registered alongside other addons. It's not doing anything that would require being registered is it? No panels/ toolbar/tabs?

For the error, seems to be working here in both dev mode and build/host for storybook. Any relevant errors in the server console?

Reply:
Oh right, sorry, yeah that's correct. No changes involving panels/toolbar/tabs, so no need to register.

The error was because I removed `addons: ["@storybook/addon-docs"]`, thinking that was what you were referring to previously, so no errors on my end!